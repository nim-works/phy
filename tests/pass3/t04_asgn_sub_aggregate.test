(TypeDefs
  (Int 8)
  (Record 8
    (Field 0 (Type 0)))
  (Record 8
    (Field 0 (Type 1)))
  (ProcTy (Void)))
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 3)
    (Locals (Type 2) (Type 1))
    (Continuations
      (Continuation (Params) (Locals (Local 0) (Local 1))
        (Asgn
          (Field (Local 0) 0)
          (Copy (Local 1)))
        (Continue 1))
      (Continuation (Params)))))
