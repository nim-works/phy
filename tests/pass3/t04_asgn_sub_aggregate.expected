;$sexp
(TypeDefs
  (Int 8)
  (Blob 8)
  (Blob 8)
  (ProcTy (Void)))
(GlobalDefs)
(ProcDefs
  (ProcDef (Type 3)
    (Locals (Type 2) (Type 1))
    (Continuations
      (Continuation (Params)
        (Locals (Local 0) (Local 1))
        (Blit
          (Add (UInt 8)
            (Addr (Local 0))
            (IntVal 0))
          (Addr (Local 1))
          (IntVal 8))
        (Continue 1))
      (Continuation (Params)))))
